modules/tensor_mechanics/src/materials/StressUpdateBase.C

```diff
@@ -31,3 +31,8 @@ StressUpdateBase::setQp(unsigned int qp)
 {
   _qp = qp;
 }
+
+void
```

Comment:
On second thought, here's a better solution: Leave the implementation in the base class, but have it call `mooseError()`, so that if anyone tries to use the cyclic iterations with a model that doesn't support it, they'll get an error.

I think we can figure out a way to have the implementation in StressUpdateBase copy all stateful properties so we won't need to implement this in derived classes, but that will work in the mean-time.

Also, while you're making that change, how about renaming `propagateQpProperties` to `propagateQpStatefulProperties`?  That would make it clear that this only applies to stateful properties.